app/analysis/cms.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+module.exports = {
+    name: 'cms',
+    description: 'Czy strona jest budowana przy pomocy narzedzie CMS ?',
+    run: function (page, callback) {
+        page.evaluate(function () {
+            var allMeta = document.querySelectorAll('meta[name="generator"]');
+            var cmsName;
+            var n;
+            if (allMeta.length === 1) {
```

Comment:
lot of nested if's, it looks scary ..
why not use array with cms names?

btw, we need test for all cases (now, there is test only for Joomla)